### PR TITLE
maint:  perform some PF4 porting - mainly Buttons

### PR DESCRIPTION
**Pull request by KKoukiou**

### CODE REVIEW

File: src/ContainerCommitModal.jsx [hunk 1]
```diff
@@ -132,8 +132,8 @@ class ContainerCommitModal extends React.Component {
                 <div key={"onbuildvar" + idx} id="select-claimed-onbuildvars" className="form-inline containers-run-onbuildvarclaim containers-run-inline">
                     <FormGroup className="form-inline">
                         <input type="text" name="onbuildvar_key" onChange={(evt) => this.handleOnBuildsInputChange(idx, evt)} />
-                        <button type="button" className="btn btn-default pficon-close" disabled={idx === 0} onClick={() => this.handleRemoveOnBuild(idx)} />
-                        <button type="button" className="btn btn-default fa fa-plus" onClick={this.handleAddOnBuild} />
+                        <Button variant="secondary" className="pficon-close" isDisabled={idx === 0} onClick={() => this.handleRemoveOnBuild(idx)} />
+                        <Button variant="secondary" className="fa fa-plus" onClick={this.handleAddOnBuild} />
                     </FormGroup>
```
Comment (marusak) [inline src/ContainerCommitModal.jsx:137]: This seems to be in a wrong commit.

File: src/Containers.jsx [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import ReactDOM from "react-dom";
+import { Button } from '@patternfly/react-core';
```
Comment (marusak) [inline src/Containers.jsx:3]: This whole file seems to be in a wrong commit.

File: src/ContainerRemoveErrorModal.jsx [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { useState } from 'react';
```
Comment (marusak) [inline src/ContainerRemoveErrorModal.jsx:1]: This commit message says 'Closes #359' which is not correct :)

**Comment by marusak** [comment]
Thanks! Just two changes were in wrong commit and one commit message was not correct. And needs 'Closes' with the correct number :)

**Comment by KKoukiou** [comment]
I also moved the empty state pattern to PF4.
Screen Shot 2020-04-24 at 19 50 53

**Comment by marusak** [comment]
```
Errors during downloading metadata for repository 'RHEL-8-NIGHTLY-BaseOS':
  - Status code: 403 for http://download.eng.bos.redhat.com/rhel-8/nightly/RHEL-8/latest-RHEL-8.2/compose/BaseOS/x86_64/os/repodata/repomd.xml 
```

unrelated. See also https://github.com/cockpit-project/bots/issues/797

**Pull request merged**